wcycle/WCYCLE-0.DATA

```diff
@@ -0,0 +1,306 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+--
+-- Copyright (C) 2018 Equinor
+--
+--
+-- boxmodel    ||   1 horizontal producer, standard well completion (no msw)
+--
+-- Basic wcycle setup
+--
+--=====================================================================
+RUNSPEC
+--=====================================================================
+NOECHO
+
+TITLE
+  boxmodel
+
+DIMENS
+-- NX  NY  NZ
+   7  18  20  /
+
+OIL
+GAS
+WATER
+DISGAS
+METRIC
+
+-----------------------------------------------------
+-- Special options
+-----------------------------------------------------
+----     Saturation table end point scaling
+ENDSCALE
+-- directional reversible? NTENDP  NSENDP
+  'NODIR'      'REVERS'       1       20  /
+
+--       Allow 'MULTX-', 'MULTY-' etc.
+GRIDOPTS
+ 'YES'  15 /
+ 
+PATHS
+ 'ECLPATH' '../../eclipse/include' /
+/
```

Comment:
Tiny nit: This isn't actually used anywhere&ndash;in any of the cases&ndash;so maybe we could remove it to avoid confusion?

Reply:
absolutely, done.